src/routes/jira/sync/jira-sync-post.test.ts

```diff
@@ -6,12 +6,17 @@ import supertest from "supertest";
 import { getLogger } from "config/logger";
 import { encodeSymmetric } from "atlassian-jwt";
 import { sqsQueues } from "~/src/sqs/queues";
+import { GitHubServerApp } from "models/github-server-app";
+import { v4 as newUUID } from "uuid";
 
 jest.mock("~/src/sqs/queues");
 
 describe("sync", () => {
 	let app: Express;
 	let installation: Installation;
+	const installationIdForCloud = 1;
+	const installationIdForServer = 2;
```

Comment:
I suggest use the same value to mimic the conflict of the gitHubInstallationId.

Reply:
For some reason, I can't create 2 separate subscriptions using the same `installationId`.

Reply:
Resolving this, as it is related to a different problem: https://atlassian.slack.com/archives/C039YPCGCSG/p1660780825781989